app/controllers/repository_rows_controller.rb

```diff
@@ -477,6 +489,10 @@ def fetch_columns_list_items
     collection
   end
 
+  def show_params
```

Comment:
What's the point in this method? These params are not used for attribute assignment